docs/Development Guidelines.md

```diff
@@ -66,4 +67,5 @@ git push origin your-branch-name-here --force
 
 ### Merging your PR
 - Once reviewers are happy, and you have rebased, you can merge using the green merge button on the PR
+- **Note: Merging should be carried out by the pull request author to reduce unintended merge behaviour or complications**
```

Comment:
It seems past me left a ticket to update the PR docs a while ago but forgot. Maybe the scare yesterday would've been averted if I did it earlier. Better late than never. I've just added a few sentences to explain bits which I think are important to know.

Reply:
Great Idea!